wiki/Help_centre/Tournament_bans/en.md

```diff
@@ -4,15 +4,15 @@
 
 A tournament ban prevents a player from playing in official or officially supported tournaments. It also limits to what extent a player can staff or otherwise offer assistance in these tournaments.
 
-Tournament bans are very rarely handed out on their own, but generally speaking almost all players who return under a standard appeal for an [account restriction](/wiki/Help_centre/Account_restrictions) are subject to one of at least a year in length.
+Tournament bans are very rarely handed out on their own, but generally speaking, almost all players who return under a standard appeal for an [account restriction](/wiki/Help_centre/Account_restrictions) are subject to one of at least a year in length.
```

Comment:
Incorrect usage of the new comma. The previous one was fine already joining two independent clauses together.

You would only use this comma sandwich to add a clause in between, sort of as you'd do with em dashes.